ansible/playbooks/local-services-cfg.yml

```diff
@@ -31,107 +31,198 @@
 
     - role: util-cfg-service
       service_name_short: "{{iplant_groups.service_name_short}}"
+      tags:
+        - service_config
+        - iplant-groups
 
     - role: util-cfg-service
       service_name_short: "{{jexevents.service_name_short}}"
+      tags:
+        - service_config
+        - jexevents
 
     - role: util-cfg-service
       service_name_short: "{{kifshare.service_name_short}}"
+      tags:
+        - service_config
+        - kifshare
 
     - role: util-cfg-service
       service_name_short: "{{metadata.service_name_short}}"
+      tags:
+        - service_config
+        - metadata
 
     - role: util-cfg-service
       service_name_short: "{{monkey.service_name_short}}"
+      tags:
+        - service_config
+        - monkey
 
     - role: util-cfg-service
       service_name_short: "{{notificationagent.service_name_short}}"
+      tags:
+        - service_config
+        - notificationagent
 
     - role: util-cfg-service
       service_name_short: "{{saved_searches.service_name_short}}"
+      tags:
+        - service_config
+        - saved-searches
 
     - role: util-cfg-service
       service_name_short: "{{terrain.service_name_short}}"
+      tags:
+        - service_config
+        - terrain
 
     - role: util-cfg-service
       service_name_short: "{{tree_urls.service_name_short}}"
+      tags:
+        - service_config
+        - tree-urls
 
     - role: util-cfg-service
       service_name_short: "{{user_preferences.service_name_short}}"
+      tags:
+        - service_config
+        - user-preferences
 
     - role: util-cfg-service
       service_name_short: "{{user_sessions.service_name_short}}"
+      tags:
+        - service_config
+        - user-sessions
 
     - role: util-cfg-service
       service_name_short: "{{templeton_periodic.service_name_short}}"
       src: "templeton-periodic.yaml.j2"
       dest: "{{service_conf_dir}}/templeton-periodic.yaml"
+      tags:
+        - service_config
+        - templeton-periodic
 
     - role: util-cfg-service
       service_name_short: "{{templeton_incremental.service_name_short}}"
       src: "templeton-incremental.yaml.j2"
       dest: "{{service_conf_dir}}/templeton-incremental.yaml"
+      tags:
+        - service_config
+        - templeton-incremental
 
     - role: util-cfg-service
       src: "ui/de-application.yaml.j2"
       dest: "{{service_conf_dir}}/de-application.yaml"
       service_name_short: "{{de.service_name_short}}"
+      tags:
+        - service_config
+        - de-ui
 
     - role: util-cfg-service
       src: "ui/de.properties.j2"
       dest: "{{service_conf_dir}}/de.properties"
       service_name_short: "{{de.service_name_short}}"
+      tags:
+        - service_config
+        - de-ui
 
     - role: util-cfg-service
       src: "ui/nginx.conf.j2"
       dest: "{{service_conf_dir}}/nginx/nginx.conf"
       service_name_short: "{{de.service_name_short}}"
+      tags:
+        - service_config
+        - de-ui
```

Comment:
I've been very lax in reviewing this, sorry about that. If we want to completely match the tags we use for deployment, this (and the other UI ones) should be `ui` rather than `de-ui`.